main.go

```diff
@@ -168,7 +190,11 @@ func main() {
 	})
 
 	mux := http.NewServeMux()
-	mux.Handle("/", r)
+	if *acceptGzip {
+		mux.Handle("/", decompressGzip()(r))
+	} else {
+		mux.Handle("/", r)
+	}
```

Comment:
So this can now be simply `mux.Handle("/", gunzipRequest(r))`.

Reply:
ok